package.json

```diff
@@ -186,9 +186,6 @@
     "@department-of-veterans-affairs/formation": "^1.18.0",
     "@department-of-veterans-affairs/react-jsonschema-form": "^1.0.0",
     "@octokit/rest": "^16.2.0",
-    "apollo-boost": "^0.1.23",
-    "apollo-cache-inmemory": "^1.3.12",
-    "apollo-link-http": "^1.5.9",
```

Comment:
While using Apollo, I ran into an issue with using a fragment in a query. This is a super basic use case, but apparently there's an outstanding bug with - https://github.com/apollographql/apollo-client/issues/3882. At that point, I lost all faith in using Apollo so I went with plain old `fetch`.

Reply:
Is there anything important we're losing from Apollo as far as optimizing the request that's made? I don't know all that Apollo does, but I assume it's doing more than just constructing a fetch call.

Reply:
I don't think it's a good use case for us. If we were querying regularly, the caching system would be helpful, or if we were interested in implementing it into React then it may also have saved us something because the docs lead to React implementation pretty often. I could easily be missing something though.